faq.md

```diff
@@ -467,7 +467,7 @@ to the mobile platform using a
 [`BasicMessageChannel`](https://docs.flutter.io/flutter/services/BasicMessageChannel-class.html).
 
 Learn more about accessing platform and third-party services in Flutter with
-[platform channels](/platform-channels/).
+[platform channels](https://flutter.io/platform-channels/).
```

Comment:
let's remove the https://flutter.io part. That makes it hard to stage an in-progress version of the site.

Reply:
Sorry, for some reason the URL checked kept failing on the the relative URL (until it didn't). I'll fix the URLs